cleaning_services/data/product_pricelist.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="product_pricelist_1" model="product.pricelist">
```

Comment:
This record (and its assignation in res_partner) can be removed (default value when activating pricelists)